File: curso python/1er _semestre/mayor3NUM.py
a=int (input('introduce el primer numero: '))
b=int (input('introduce el segundo numero: '))
c=int (input('introduce el tercer numero: '))

if a>b:
    if a>c:
        m=a
    else:
        m=c
else:
    if b>c:
        m=b
    else:
        m=c
print ('el mayor de los 3 numeros es: ',m)

File: curso c++/semestre/mayor3_num.cpp
#include <iostream>
using namespace std;
int main (){
    //mayor de 3 numeros 
    //variables
    int a,b,c,m;
    //entrada
    cin>>a>>b>>c;
    if (a>b)
        if(a>c)
            m=a;
        else
            m=c;
    else
        if(b>c)
            m=b;
        else
            m=c;       
    cout <<"MAYOR DE LOS 3 NUMEROS ES: "<< m;
    
    return 0;
}